### ISSUE TITLE
ArmadilloHTTP.h:5:24: fatal error: AardvarkTCP.h: No such file or directory

### ISSUE THREAD

**Issue by lpotter** [philbowles/ArmadilloHTTP#1]
It may be a missing name change commit in  AardvarkTCP, but the header file there is named H4AsyncTCP.h

Or am I missing something?

**Comment by kebasaa** [comment]
You are right about this, some of H4 is broken. But you can install H4 with modified libraries, which will word: [https://github.com/kebasaa/h4installer](https://github.com/kebasaa/h4installer)
